Lib/test/test_sys.py

```diff
@@ -1179,8 +1181,8 @@ class X(Exception):
                 with test.support.captured_stderr() as stderr, \
                      test.support.swap_attr(sys, 'unraisablehook',
                                             sys.__unraisablehook__):
```

Comment:
Having the continuation align with the indent makes this a bit misleading. Maybe increase the indent?

```suggestion
                         test.support.swap_attr(sys, 'unraisablehook',
                                                sys.__unraisablehook__):
```

Reply:
I ran the code block through black to make it less ugly.